src/components/sidebar/sidebar.css

```diff
@@ -164,7 +164,7 @@
   height: 0;
   border-width: 5px 4px 5px 8px;
   border-style: solid;
-  border-color: transparent transparent transparent currentcolor;
+  border-color: transparent transparent transparent currentColor;
```

Comment:
Nice catch! Thanks stylelint! 😄

Reply:
this is just for consistency :-) Not really a bug !

Reply:
Ah, right. I keep forgetting that colors are case-insensitive 😄